_data/hosting_providers.json

```
@@ -2298,5 +2298,15 @@
     "plan": "",
     "reviewed": "2020.01.29",
     "note": "Shared Web Hosting utilizes Plesk Panel interface to enable Let's Encrypt; Managed Cloud Hosting enabled via Certbot on request."
+  },
+    {
+    "name": "CA Net, Europe)",
```

Did you mean to include this trailing parenthesis?
```suggestion
    "name": "CA Net, Europe",
```